frontends/ios/BitBoxApp/BitBoxApp/WebView.swift

```diff
@@ -156,6 +158,16 @@ struct WebView: UIViewRepresentable {
     }
     
     class Coordinator: NSObject, WKNavigationDelegate, WKUIDelegate {
+        func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+            // change alpha back to 1.0 after a slight delay
+            // this is to prevent the white flash (flicker) on initial load
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.4) {
+                UIView.animate(withDuration: 0.4) {
+                    webView.alpha = 1.0
+                }
+            }
```

Comment:
This fixes the loading time of the app to 0.8s. This could be too long for some devices and too short for others, right?

Maybe we can have a smarter loading screen. What is the exact cause of the white flicker?